### ISSUE TITLE
PlaceProxy

### ISSUE THREAD

**Issue by hainm** [arose/nglview#477]
```html
%%html
<div class='foobar'></div>
```

```python
from ipywidgets import PlaceProxy
PlaceProxy(child=view, selector='.foobar')
```

note for myself.

**Comment by hainm** [comment]
![picture5](https://cloud.githubusercontent.com/assets/4451957/18222561/fceb8156-7168-11e6-8e07-d07b9c75f4e6.png)

**Comment by hainm** [comment]
this is very helpful. I can exclude most of my code for 'DraggableBox'

- create a placeholder for `NGLWidget`
- Use `PlaceProxy` to put any ipywidgets's Widget to NGLWidget.

**Comment by hainm** [comment]
implemented in #481
